receiver/vcenterreceiver/scraper.go

```diff
@@ -349,12 +342,6 @@ func (v *vcenterMetricScraper) collectVMs(
 			}
 		}
 
-		hostname, err := vmHost.ObjectName(ctx)
```

Comment:
minor and wouldn't hold up PR: since hostname is references a few times, it's still probably worth it to make a variable out of it after creating the `hwSum` variable